ppl/src/main/java/org/opensearch/sql/ppl/parser/AstBuilder.java

```diff
@@ -121,7 +123,10 @@ public UnresolvedPlan visitSearchFilterFrom(SearchFilterFromContext ctx) {
   @Override
   public UnresolvedPlan visitDescribeCommand(DescribeCommandContext ctx) {
     final Relation table = (Relation) visitTableSourceClause(ctx.tableSourceClause());
-    return new Relation(qualifiedName(mappingTable(table.getTableName())));
+    QualifiedName tableQualifiedName = table.getTableQualifiedName();
+    ArrayList<String> parts = new ArrayList<>(tableQualifiedName.getParts());
+    parts.set(parts.size() - 1, mappingTable(parts.get(parts.size() - 1)));
```

Comment:
Sorry what does this mean? No UT changes required to cover this?

Reply:
Will make comments and also include UTs.

Reply:
should it be handled by catalog resolver?

Reply:
yes, CatalogSchemaIdentifierNameResolver can handle this.

Reply:
Future Design
describe prometheus.requests_total --> source = prometheus.information_schema.columns.

information_schema will be a separate information_schema internal connector which can interact with all the storage engines to get column and table details.

Storage Engine will have more methods in interface to expose tables, columns, schema